rootfs/templates/wrapper-body.sh

```
@@ -0,0 +1,572 @@
+
+if [ "${GEODESIC_SHELL}" == "true" ]; then
+	echo "Cannot run while in a geodesic shell"
+	exit 1
+fi
+
+require_installed tr
+require_installed grep
+require_installed docker
+
+docker ps >/dev/null 2>&1
+if [ $? -ne 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)